firefox-ios/Client/Frontend/Browser/BrowserViewController/State/BrowserViewControllerState.swift

```diff
@@ -160,35 +158,6 @@ struct BrowserViewControllerState: ScreenState, Equatable {
             microsurveyState: MicrosurveyPromptState.reducer(state.microsurveyState, action))
     }
 
-    static func reduceStateForPrivateModeAction(action: PrivateModeAction,
-                                                state: BrowserViewControllerState) -> BrowserViewControllerState {
-        switch action.actionType {
-        case PrivateModeActionType.privateModeUpdated:
```

Comment:
An update is now triggered every time the a tab is selected so BVC no longer needs to listen for this action.